### PR TITLE
Resolve issue of execution stuck when executing with core-command

### PR DESCRIPTION
**Pull request by bhavikkothari-eaton**
Resolve Issue of Execution stuck while running core-command

### CODE REVIEW

File: prerequisite_changes.py [hunk 1]
```diff
@@ -119,8 +123,6 @@ def main():
     general_changes()
     if service == "core-metadata":
         # This is core-metadata service specific changes
```
Comment (rajatkumar-eaton) [inline prerequisite_changes.py:125]: @bhavikkothari-eaton need to remove this comment also?

File: prerequisite_changes.py [hunk 2]
```diff
@@ -174,8 +176,6 @@ def main():
         print("This is core-data service related changes")
     if service == "core-command":
         # This is core-metadata service specific changes
```
Comment (rajatkumar-eaton) [inline prerequisite_changes.py:178]: @bhavikkothari-eaton need to remove this comment also?

Comment (bhavikkothari-eaton) [reply prerequisite_changes.py:178]: we can keep this comment, as in this block changes are specific to services.

File: prerequisite_changes.py (hunk 1 above)
Comment (bhavikkothari-eaton) [reply prerequisite_changes.py:125]: we can keep this comment

**Review by rajatkumar-eaton — APPROVED** [review]

**Comment by rajatkumar-eaton** [comment]
- [x] @NileshDixit-eaton Please validate and confirm against https://jira-prod.tcc.etn.com/browse/LNXTK-21503

**Review by rajatkumar-eaton — APPROVED** [review]

**Pull request merged**